### ISSUE TITLE
Need new version of active_attr for Rails 5.2.0.alpha

### ISSUE THREAD

**Issue by jasnow** [cgriego/active_attr#158]
Need new version of active_attr for edge ("master") Rails (5.2.0.alpha).
Currently getting:
```
Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    active_attr was resolved to 0.1.0, which depends on
      activesupport (~> 3.1)

    factory_girl_rails was resolved to 4.8.0, which depends on
      factory_girl (~> 4.8.0) was resolved to 4.8.0, which depends on
        activesupport (>= 3.0.0)

    rails was resolved to 5.2.0.alpha, which depends on
      activejob (= 5.2.0.alpha) was resolved to 5.2.0.alpha, which depends on
        globalid (>= 0.3.6) was resolved to 0.3.7, which depends on
          activesupport (>= 4.1.0)

    rails was resolved to 5.2.0.alpha, which depends on
      activesupport (= 5.2.0.alpha)
```

**Comment by yangzhichina** [comment]
I have the same issue on the Rails 5.1.0.

**Comment by cgriego** [comment]
I've released ActiveAttr 0.10.1 which is compatible with Rails 5.1 and allows 5.2 prereleases.

**Comment by jasnow** [comment]
Oh - okay - THANKS!!!
